### PR TITLE
finished video block

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/js/templates/VideoBlock.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+})
+
+fullScreenButton.addEventListener('click', () => {
+    if (fullScreenButton.classList.contains('animated')) {
+        fullScreenButton.classList.remove('animated')
+        animationFrom.beginElement()
+        setTimeout(() => {
+            videoBlockContainer.classList.remove('full')
+            document.body.removeAttribute("style")
+        }, 300)
```
Comment (StadniukYevhen) [inline src/js/templates/VideoBlock.js]: create variable with delay value

File: src/js/templates/VideoBlock.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    videoProgressCircle.style.left = value
+
+    if (videoProgress.style.width != '100%') {
+        requestAnimationFrame(updateProgress)
+    }
+});
+
+//when video timebar clicked
+seekBar.addEventListener('mousedown', function(e) {
+    let position = e.pageX - seekBar.getBoundingClientRect().left
```
Comment (StadniukYevhen) [inline src/js/templates/VideoBlock.js]: rewrite to `conts`

File: src/js/templates/VideoBlock.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    }
+    videoProgress.setAttribute('width', percentage + '%')
+    videoBlockPlayer.currentTime = videoBlockPlayer.duration * percentage / 100
+})
+
+// changeVideo
+videoItemControler.forEach((element, index) => {
+    element.addEventListener('click', () => {
+        videoBlockPlayer.src = `videos/q7-video-block${index+1}.mp4`
+        videoBlockPlayer.poster = `css/images/q7-video-block${index+1}.jpg`
```
Comment (StadniukYevhen) [inline src/js/templates/VideoBlock.js]: add spaces around `+`

File: src/js/templates/VideoBlock.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    if (videoProgress.style.width != '100%') {
+        requestAnimationFrame(updateProgress)
+    }
+});
+
+//when video timebar clicked
+seekBar.addEventListener('mousedown', function(e) {
+    let position = e.pageX - seekBar.getBoundingClientRect().left
+    let percentage = 100 * position / seekBar.offsetWidth
+    if(percentage > 100) {
```
Comment (StadniukYevhen) [inline src/js/templates/VideoBlock.js]: rewrite using `&&` or `ternary` operator

File: src/js/templates/VideoBlock.js (hunk 3 above)
Comment (nmelnychenko) [reply src/js/templates/VideoBlock.js]: done

File: src/js/templates/VideoBlock.js (hunk 2 above)
Comment (nmelnychenko) [reply src/js/templates/VideoBlock.js]: done

File: src/js/templates/VideoBlock.js (hunk 3 above)
Comment (nmelnychenko) [reply src/js/templates/VideoBlock.js]: `let percentage = 100 * position / seekBar.offsetWidth;
    ( percentage > 100 ) && ( percentage = 100 );
    ( percentage < 0 ) && ( percentage = 0 );`
I need to choose what to use here:
- if() statement
or
- `;` symbol

File: src/js/templates/VideoBlock.js (hunk 1 above)
Comment (nmelnychenko) [reply src/js/templates/VideoBlock.js]: done

**Pull request merged**